gammapy/data/data_store.py

```diff
@@ -676,18 +679,21 @@ def __str__(self):
         return s
 
     def make_psf(self, position, energy=None, theta=None):
-        """Make energy-dependent mean PSF for a given position and a set of observations.
+        """Make energy-dependent mean PSF for a given position and a set of
+        observations.
 
         Parameters
         ----------
         position : `~astropy.coordinates.SkyCoord`
             Position at which to compute the PSF
         energy : `~astropy.units.Quantity`
             1-dim energy array for the output PSF.
-            If none is given, the energy array of the PSF from the first observation is used.
+            If none is given, the energy array of the PSF from the first
+            observation is used.
         theta : `~astropy.coordinates.Angle`
             1-dim offset array for the output PSF.
-            If none is given, the energy array of the PSF from the first observation is used.
+            If none is given, the energy array of the PSF from the first
+             observation is used.
```

Comment:
Maybe this line is causing the Sphinx / RST error?
Try removing a space here, i.e. indent these two lines exactly the same:
```
If none is given, the energy array of the PSF from the first
 observation is used.
```